text/0000-afp.md

```diff
@@ -0,0 +1,354 @@
+# ARA FARMING PROTOCOL
+
+---
+Request For Comments: 0000
```

Comment:
I think this number should be 0005 ?  or is it only updated when the RFC is adopted (asking for a friend).

Reply:
Only updated after it's approved :)